keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -631,17 +630,35 @@ _This section must be completed when targeting beta graduation to a release._
   Try to be as paranoid as possible - e.g., what if some components will restart
    mid-rollout?
 
+If the CRI implementation doesn't support the required metrics, and cAdvisor has container metrics collection turned off,
+it is possible the node comes up with no metrics about pods and containers. This should be mitigated by making sure that
+the kubelet probes the CRI implementation and enables cAdvisor metrics collection even if the feature gate is on.
+
 * **What specific metrics should inform a rollback?**
 
+The lack of any metrics reported for pods and containers is the worst case scenerio here, and would require a rollback.
```

Comment:
Is it sufficient to disable the feature flag in a rollback scenario?

Reply:
yes good point, I will add this